backend/src/federation/index.ts

```diff
@@ -4,11 +4,22 @@
 import DHT from '@hyperswarm/dht'
 // import { Connection } from '@dbTools/typeorm'
 import { backendLogger as logger } from '@/server/logger'
+import CONFIG from '@/config'
 
 function between(min: number, max: number) {
   return Math.floor(Math.random() * (max - min + 1) + min)
 }
 
+const getSeed = (): Buffer | null => {
+  if (CONFIG.FEDERATION_DHT_SEED) {
+    logger.debug(`with seed='${CONFIG.FEDERATION_DHT_SEED}'`)
+    return Buffer.alloc(KEY_SECRET_SEEDBYTES, CONFIG.FEDERATION_DHT_SEED)
+  }
+  return null
+}
+
+const KEY_SECRET_SEEDBYTES = 32
```

Comment:
funny that this is working - but it must be defined before used - i think that would be more safe and appropriate. Move above the `getSeed()` function.

Reply:
done